RustConfig

```diff
@@ -0,0 +1 @@
+VERSION=nightly-2018-11-29
```

Comment:
Could you explain the difference in RustConfig vs. rust-toolchain? I couldn't find much info online about what uses the former.